src/pip/_internal/network/auth.py

```diff
@@ -23,11 +20,10 @@
 from pip._internal.utils.typing import MYPY_CHECK_RUNNING
 
 if MYPY_CHECK_RUNNING:
-    from optparse import Values
-    from typing import Dict, Optional, Tuple
+    from typing import Dict, Optional, Tuple, List, Any
 
     from pip._internal.vcs.versioncontrol import AuthInfo
-
+    from pip._vendor.requests.models import Response, Request
```

Comment:
nit: visual separation of imports

```suggestion

    from pip._vendor.requests.models import Response, Request

```

Reply:
Fixed in latest commit.